tests/e2e/utils/helpers.go

```diff
@@ -79,6 +79,14 @@ func GetAPMDir() string {
 	return path.Join(usr.HomeDir, constants.APMDir)
 }
 
+func GetSnapshotsDir() string {
```

Comment:
can we use app obtained from GetApp() for this paths?

Reply:
updated `baseDir` const in `tests/e2e/utils/constants.go` and we don't need this change anymore. all existing logic simply work